app/assets/stylesheets/forever_style_guide/modules/_nav-account_dropdown.scss

```diff
@@ -80,9 +80,15 @@ $padding-small-horizontal: 3px;
 .navigation-account_dropdown {
   @media (min-width: $grid-float-breakpoint) {
     @include off-canvas-style;
+    border-left: solid 1px $color-gray-200;
+    box-shadow: 0 3px 5px rgba(0, 0, 0, 0.1);
   }
 }
 
+.navigation-account_dropdown.in {
```

Comment:
rather than add styling, we can handle this by upating the shared mixin from scroll to auto: https://github.com/forever-inc/forever-style-guide/blob/dev/app/assets/stylesheets/forever_style_guide/globals/_mixins.scss#L64